events/mock_event.json

```diff
@@ -0,0 +1,135 @@
+{
+  "resource": "/{proxy+}",
```

Comment:
I recall putting those events directly from the webconsole, I wonder if those could be trimmed down a little bit more without impacting functionality, @victorskl?

Reply:
All `mock_event*.json` are for local dev server purpose only; i.e. it is how `cargo lambda start` work in local. So, don't think trimming down will be matter much. But, yah, pls feel free to give it a try trim there. 

> I recall putting those events directly from the webconsole

Yep, pls don't use these mock event JSON in Lambda webconsole. Just use those with `event*.json`. That's the actual HTTP POST payload for the application need.